Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.yml

```diff
@@ -0,0 +1,1824 @@
+      description: 'The folder where the file resides. This attribute does not include the name of the file. Example:c:\windows\system32\.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.last_seen
+      description: 'The timestamp (in ISO 8601 format) that specifies the creation time of the event that resulted into the creation of this association. Example: YYYY-MM-DDTHH:MM:SS.sssZ.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.name
+      description: 'The name of the file. This attribute does not include the path of the file. Example: sc.exe.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.sha2
+      description: 'The SHA256 checksum of the file (hex string). Example: eaab690ebd8ddf9ae452de1bc03b73c8154264dbd7a292334733b47a668ebf31.'
+      type: String
+  - arguments:
+    - default: false
+      description: |-
+        The limit for the number of events listed per page.
+        Default is '50'.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        The page number to view. Each page contains page_size values. Must be used along with page_size.
+        Default is '1'.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: The number of results per page to display.
+      isArray: false
+      name: page_size
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Specify a search query as a Lucene query string.
+        Example: query="external_ip: 8.8.8.8"
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    deprecated: false
+    description: Get domain instances.
+    execution: false
+    name: symantec-edr-domain-instance-list
+    outputs:
+    - contextPath: SymantecEDR.DomainInstance.data_source_url
+      description: 'Last URL accessed on this domain. Example: http://www.<domain>.com/shample/shample.exe.'
+      type: String
+    - contextPath: SymantecEDR.DomainInstance.data_source_url_domain
+      description: 'The name of the domain. Example: skyscan.com.'
+      type: String
+    - contextPath: SymantecEDR.DomainInstance.disposition
+      description: 'Domain disposition: 0 = healthy/good, 1 = unknown, 2 = suspicious, 3 = bad.'
+      type: Number
+    - contextPath: SymantecEDR.DomainInstance.frst_seen
+      description: 'The timestamp (in ISO 8601 format) that specifies the creation time of the event that resulted into the creation of this instance. Example: YYYY-MM-DDTHH:MM:SS.sssZ.'
+      type: String
+    - contextPath: SymantecEDR.DomainInstance.last_seen
+      description: 'The timestamp (in ISO 8601 format) that specifies the creation time of the event that resulted into the update of this instance. Example: YYYY-MM-DDTHH:MM:SS.sssZ.'
```

Comment:
Why is this description the same as the one for first_seen?  It should be different,